site/docs/v1/tech/guides/account-management.adoc

```diff
@@ -0,0 +1,190 @@
+---
+layout: doc
+title: Account Management
+description: Managing Accounts
+---
+:page-liquid:
+
+include::docs/v1/tech/shared/_premium-edition-blurb.adoc[]
+
+[NOTE.since]
+====
+Available since 1.26.0
+====
+
+== Overview
+
+This allows you the manage accounts.
+
+* <<Setup MFA as Admin (from UI)>>
```

Comment:
I wonder if we should have a style guide that describes how to talk about MFA vs Two-Factor and what the differences are? 

We have Two-Factor APIs and Two-Factor methods, but in a larger sense we support Multi-Factor authentication because we allow the user to configure one to many methods (multi). 

But the actually configuration is to allow the user to enable or or more Two-Factor methods, if we want to talk about these configuration steps as MFA that is fine, but we should come up with a strategy to make sure we can be consistent between APIs, UI and the panel text, etc.

Reply:
Opened an issue on the internal board to track.  Can incorporate into this PR, perhaps best to draft it in a separate one.